eclipse-scout-core/src/tree/Tree.ts

```diff
@@ -3056,6 +3065,10 @@ export class Tree extends Widget implements TreeModel {
   }
 
   protected _installNodes(nodes: TreeNode[]) {
+    this._updateNodeSize(nodes);
+  }
+
+  protected _updateNodeSize(nodes: TreeNode[]) {
```

Comment:
Maybe name it updateNodeSizes (plural) because it takes an array of nodes. Also please move it up to the other update size related methods (updateNodeHeights, _updateNodeDimensions).